crates/file-explorer/src/fs.rs

```diff
@@ -10,6 +10,7 @@ use axum::body::Bytes;
 use chrono::{DateTime, Local};
 use futures::Stream;
 use mime_guess::{from_path, Mime};
+#[allow(unused_imports)]
```

Comment:
Please remove the unused import if it is complaining

```suggestion
```

Reply:
I don't know why there is dead code in the crate file-explorer but the warnings are present.